test/integration/environment/setup.js

```diff
@@ -15,16 +17,87 @@ const API_KEY_USER_ENDPOINT = '/builtin-users/dataverseAdmin/api-token';
 const API_KEY_USER_PASSWORD = 'admin1';
 
 module.exports = async () => {
+  await setupContainers();
+  await setupApiKey();
+  await setupTestFixtures();
+};
+
+async function setupContainers() {
```

Comment:
I ran the integration tests and the first time it went great, but the second time I got this error:

```
(base) melina@MacBook-Air-de-Melina dataverse-client-javascript % npm run test:integration

> js-dataverse@2.0.0 test:integration
> jest -c jest.config.integration.js

RUNNING INTEGRATION TESTS
Determining test suites to run...Cleaning up old container volumes...
Running test containers...
Test containers up and running
Obtaining test API key...
API key obtained
Creating test datasets...
Test datasets created
Waiting for datasets indexing in Solr...
Tests setup: Error while waiting for datasets indexing in Solr: [400] Search Syntax Error: Error from server at http://solr:8983/solr/collection1: SolrCore 'collection1' is not available due to init failure: Could not load conf for core collection1: Can't load schema null: Error fetching schema
Tests setup: Error while waiting for datasets indexing in Solr: [400] Search Syntax Error: Error from server at http://solr:8983/solr/collection1: SolrCore 'collection1' is not available due to init failure: Could not load conf for core collection1: Can't load schema null: Error fetching schema
Tests setup: Error while waiting for datasets indexing in Solr: [400] Search Syntax Error: Error from server at http://solr:8983/solr/collection1: SolrCore 'collection1' is not available due to init failure: Could not load conf for core collection1: Can't load schema null: Error fetching schema
Tests setup: Error while waiting for datasets indexing in Solr: [400] Search Syntax Error: Error from server at http://solr:8983/solr/collection1: SolrCore 'collection1' is not available due to init failure: Could not load conf for core collection1: Can't load schema null: Error fetching schema
Tests setup: Error while waiting for datasets indexing in Solr: [400] Search Syntax Error: Error from server at http://solr:8983/solr/collection1: SolrCore 'collection1' is not available due to init failure: Could not load conf for core collection1: Can't load schema null: Error fetching schema
Tests setup: Error while waiting for datasets indexing in Solr: [400] Search Syntax Error: Error from server at http://solr:8983/solr/collection1: SolrCore 'collection1' is not available due to init failure: Could not load conf for core collection1: Can't load schema null: Error fetching schema
Tests setup: Error while waiting for datasets indexing in Solr: [400] Search Syntax Error: Error from server at http://solr:8983/solr/collection1: SolrCore 'collection1' is not available due to init failure: Could not load conf for core collection1: Can't load schema null: Error fetching schema
Tests setup: Error while waiting for datasets indexing in Solr: [400] Search Syntax Error: Error from server at http://solr:8983/solr/collection1: SolrCore 'collection1' is not available due to init failure: Could not load conf for core collection1: Can't load schema null: Error fetching schema
Tests setup: Error while waiting for datasets indexing in Solr: [400] Search Syntax Error: Error from server at http://solr:8983/solr/collection1: SolrCore 'collection1' is not available due to init failure: Could not load conf for core collection1: Can't load schema null: Error fetching schema
Tests setup: Error while waiting for datasets indexing in Solr: [400] Search Syntax Error: Error from server at http://solr:8983/solr/collection1: SolrCore 'collection1' is not available due to init failure: Could not load conf for core collection1: Can't load schema null: Error fetching schema
Error: Jest: Got error running globalSetup - /Users/melina/GitHub/dataverse-client-javascript/test/integration/environment/setup.js, reason: Tests setup: Timeout reached while waiting for datasets indexing in Solr
    at waitForDatasetsIndexingInSolr (/Users/melina/GitHub/dataverse-client-javascript/test/integration/environment/setup.js:94:11)
    at async setupTestFixtures (/Users/melina/GitHub/dataverse-client-javascript/test/integration/environment/setup.js:63:3)
    at async module.exports (/Users/melina/GitHub/dataverse-client-javascript/test/integration/environment/setup.js:22:3)
    at async /Users/melina/GitHub/dataverse-client-javascript/node_modules/@jest/core/build/runGlobalHook.js:109:13
    at async waitForPromiseWithCleanup (/Users/melina/GitHub/dataverse-client-javascript/node_modules/@jest/transform/build/ScriptTransformer.js:160:5)
    at async ScriptTransformer.requireAndTranspileModule (/Users/melina/GitHub/dataverse-client-javascript/node_modules/@jest/transform/build/ScriptTransformer.js:805:16)
    at async runGlobalHook (/Users/melina/GitHub/dataverse-client-javascript/node_modules/@jest/core/build/runGlobalHook.js:101:9)
    at async runJest (/Users/melina/GitHub/dataverse-client-javascript/node_modules/@jest/core/build/runJest.js:327:5)
    at async _run10000 (/Users/melina/GitHub/dataverse-client-javascript/node_modules/@jest/core/build/cli/index.js:344:7)
    at async runCLI (/Users/melina/GitHub/dataverse-client-javascript/node_modules/@jest/core/build/cli/index.js:199:3)

```